go.mod

```diff
@@ -1,19 +1,22 @@
-module github.com/speedrun-website/leaderboard-backend
+module speedrun.website
```

Comment:
I generally recommend using a `{namespace}/{project}/{module}` or `{domain}/{repo} for `go mod init` e.g. the existing `github.com/speedrun-website/leaderboard-backend` or something like `speedrun.website/leaderboard-backend`

Reply:
I didn't pay attention to the module name, I just wrote something short. Switching back to `github.com/speedrun-website/leaderboard-backend`.